### PR TITLE
feat: adds openstack-exporter for prometheus

### PR DESCRIPTION
**Pull request by nicholaskuechler**

### CODE REVIEW

File: apps/appsets/components.yaml [hunk 1]
```diff
@@ -188,6 +188,27 @@ spec:
...
+                      targetRevision: '{{index .metadata.annotations "uc_deploy_ref"}}'
+                      ref: secrets
+                      path: 'secrets/{{.name}}/openstack-exporter'
+                    - repoURL: registry.scs.community/openstack-exporter
+                      chart: prometheus-openstack-exporter
+                      targetRevision: 0.4.5
+                      helm:
+                        releaseName: prometheus-openstack-exporter
+                        valueFiles:
+                          - $understack/components/openstack-exporter/values.yaml
```
Comment (cardoe) [inline apps/appsets/components.yaml:209]: If we don't need this file then just drop it.

File: apps/appsets/components.yaml [hunk 2]
```diff
@@ -188,6 +188,27 @@ spec:
...
                       targetRevision: '{{index .metadata.annotations "uc_repo_ref"}}'
                       path: 'components/chrony'
+                - component: openstack-exporter
+                  componentNamespace: monitoring
+                  skipComponent: '{{has "openstack-exporter" ((default "[]" (index .metadata.annotations "uc_skip_components") | fromJson))}}'
+                  sources:
+                    - repoURL: '{{index .metadata.annotations "uc_repo_git_url"}}'
+                      targetRevision: '{{index .metadata.annotations "uc_repo_ref"}}'
+                      path: 'components/openstack-exporter'
+                      ref: understack
```
Comment (cardoe) [inline apps/appsets/components.yaml:198]: Since we're not using anything from here just drop this section.

File: apps/appsets/components.yaml [hunk 3]
```diff
@@ -188,6 +188,27 @@ spec:
...
+                    - repoURL: '{{index .metadata.annotations "uc_deploy_git_url"}}'
+                      targetRevision: '{{index .metadata.annotations "uc_deploy_ref"}}'
+                      ref: secrets
+                      path: 'secrets/{{.name}}/openstack-exporter'
```
Comment (cardoe) [inline apps/appsets/components.yaml:202]: Is there actually any secrets needing to be applied? Or this is just the ref for the helm-config? Cause if its just the ref then drop the path.

**Review by cardoe — CHANGES_REQUESTED** [review]

File: apps/appsets/components.yaml (hunk 1 above)
Comment (nicholaskuechler) [reply apps/appsets/components.yaml]: We'll need it in the future, but I've removed it for now.

File: apps/appsets/components.yaml (hunk 2 above)
Comment (nicholaskuechler) [reply apps/appsets/components.yaml]: We'll need it in the future, but I've removed it for now.

File: apps/appsets/components.yaml (hunk 3 above)
Comment (nicholaskuechler) [reply apps/appsets/components.yaml]: We'll need it in the future, but I've removed it for now.

**Review by cardoe — APPROVED** [review]

**Pull request merged**